float32 setF
---
float32 newResult
float32 cur
float32 itg
float32 drv